documentation/assemblies/upgrading/assembly-upgrade-kafka-versions.adoc

```diff
@@ -6,9 +6,10 @@
 
 = Upgrading Kafka
 
-After you have upgraded your Cluster Operator, you can upgrade your brokers to a higher supported version of Kafka.
+After you have upgraded your Cluster Operator, you can upgrade all Kafka brokers to a higher supported version of Kafka.
 
-Kafka upgrades are performed using the Cluster Operator. How the Cluster Operator performs an upgrade depends on the differences between versions of:
+Kafka upgrades are performed using the Cluster Operator. 
+How the Cluster Operator performs an upgrade depends on the differences between the versions of:
```

Comment:
The sentences on line 18 and 20 are not valid anymore. The current state is:
* When Inter-broker protocol and Log-message format are configured in .spec.kafka.config, only one rolling update will happen.
* When they are not configured (or when only one of them is configured), two rolling updates will happen.

Reply:
Not done. I'd rather change this in a subsequent PR for multi-version upgrades.